tests/beta/queuing/test_beta_measure.py

```diff
@@ -30,10 +32,13 @@
     probs,
     Expectation,
     Sample,
+    State,
     Variance,
     Probability,
     MeasurementProcess
 )
+from pennylane.beta.tapes.qnode import QuantumFunctionError as QuantumFunctionErrorBeta  # TODO
+from pennylane.beta.queuing import state as qstate
```

Comment:
any reason for using `qstate`? A bit confusing when reading the tests to see it

Reply:
Right! I've fixed to just be `state`